packages/@aws-cdk/aws-appsync/test/appsync-union-types.test.ts

```diff
@@ -0,0 +1,152 @@
+import '@aws-cdk/assert/jest';
+import * as cdk from '@aws-cdk/core';
+import * as appsync from '../lib';
+import * as t from './scalar-type-defintions';
+
+const out = 'type Test1 {\n  test1: String\n}\ntype Test2 {\n  test2: String\n}\nunion UnionTest = Test1 | Test2\n';
+const test1 = new appsync.ObjectType('Test1', {
+  definition: { test1: t.string },
+});
+const test2 = new appsync.ObjectType('Test2', {
+  definition: { test2: t.string },
+});
+let stack: cdk.Stack;
+let api: appsync.GraphqlApi;
```

Comment:
why not put all of this into a `beforeAll()` ?

Reply:
is there a difference between giving it global scope and using beforeAll()?

Reply:
https://stackoverflow.com/questions/47913168/in-jest-are-defining-global-variables-the-same-as-if-they-are-defined-in-before

since i dont have any asynchronous code before the testing I think it's okay to put it in global scope. wdyt?